src/NUnitTestAdapter/AdapterSettings.cs

```diff
@@ -123,6 +138,45 @@ public void Load(string settingsXml)
 #endif
         }
 
+        public void SaveRandomSeed(string dirname)
+        {
+            TextWriter writer = null;
+
+            try
+            {
+                writer = new StreamWriter(Path.Combine(dirname, "$RANDOM_SEED$"));
+                writer.Write(RandomSeed.Value);
+            }
+            catch(Exception ex)
+            {
+                _logger.Error("Failed to save random seed.", ex);
+            }
+            finally
```

Comment:
A using statement probably would have been cleaner than disposing in a finally block, but good enough.